pkg/pipelines/bootstrap.go

```diff
@@ -54,7 +80,10 @@ func Bootstrap(quayUsername, baseRepo, prefix string) error {
 	}
 	defer f.Close()
 
-	dockerSecret, err := createDockerConfigSecret("regcred", f)
+	//
+	//  Create Docker Secret
+	//
+	dockerSecret, err := createDockerConfigSecret(dockerSecretName, f)
```

Comment:
Can we drop these comments and maybe make the function names better if they're not clear?